server/app/filters/UnsupportedBrowserFilter.java

```diff
@@ -0,0 +1,39 @@
+package filters;
+
+import java.util.Optional;
+import play.libs.streams.Accumulator;
+import play.mvc.Call;
+import play.mvc.EssentialAction;
+import play.mvc.EssentialFilter;
+import play.mvc.Http;
+import play.mvc.Results;
+
+/**
+ * Filter that redirects requests from unsupported browsers (e.g. IE11) to a page that tells user to
+ * use newer browser.
+ */
+public class UnsupportedBrowserFilter extends EssentialFilter {
```

Comment:
Not blocking: Consider preemptively marking this as final to make the choice to allow subclassing explicit rather than implicit.

Reply:
Done.